modules.d/35network-manager/module-setup.sh

```diff
@@ -38,6 +38,12 @@ install() {
         inst "$dbussystem"/org.freedesktop.NetworkManager.conf
         inst_multiple nmcli nm-online
 
+        # teaming support under systemd+dbus
+        inst_multiple -o                          \
+           "$systemdsystemunitdir"/teamd@.service \
+           "$dbussystem"/teamd.conf               \
+           "$dbussystemconfdir"/teamd.conf
```

Comment:
@jpirko - the fedora rpm currently defines the dbus conf in `/etc/dbus-1/system.d/teamd.conf`. Should it be modified to place it in `/usr/share/dbus-1/system.d/teamd.conf` instead?

Here I'm picking it up from either place in case it changes.